exclude_files.txt

```diff
@@ -1 +0,0 @@
-.git*
```

Comment:
Why was this removed?

Reply:
According to our standard practices, we are not including these files in any of the apps. Hence, removed this file.

Reply:
Can you explain why? As I understand, these are build exclusions similar to a manifest file. What's the reasoning for including, e.g. Git files in the app build?